lgc/builder/BuilderImpl.h

```diff
@@ -513,6 +513,14 @@ class InOutBuilder : virtual public BuilderImplBase {
   llvm::Instruction *CreateWriteBuiltInOutput(llvm::Value *valueToWrite, BuiltInKind builtIn, InOutInfo outputInfo,
                                               llvm::Value *vertexIndex, llvm::Value *index) override final;
 
+  // Create a read of (part of) a task payload.
+  llvm::Value *CreateReadTaskPayload(llvm::Type *resultTy, llvm::Value *byteOffset,
```

Comment:
The linter is complaining because these functions start with a capital "C".  You can might need to add "// NOLINT" to the end of the line or we can change it to match the code style guidelines.